benchmark/cluster_containers_setup.py

```diff
@@ -129,8 +129,8 @@ def main(reactor, argv, environ):
     # output
     eliot_to_stdout(MESSAGE_FORMATS, ACTION_START_FORMATS)
 
+    options = ContainerOptions()
     try:
-        options = ContainerOptions()
```

Comment:
I use IntelliJ, it has its own code inspection.
I do not run it for entire project because, I guess, I will get a lot of warnings, but I do look at inspection warnings for the files I modify.
When I learn code and know which parts are more important, I will run inspection on them.